lib/texture.js

```diff
@@ -201,7 +203,7 @@ function classString (x) {
 }
 
 function isCanvasElement (object) {
```

Comment:
Does it seem reasonable to move this condition to an additional function called `isOffscreenCanvas` and work that into the downstream logic, e.g. `isCanvasElement(data) || isOffscreenCanvas(data)`? Sorry to nitpick, but I just want to make sure to explicitly handle that case since perhaps an offscreen canvas shouldn't actually return true for `isCanavsElement`.

Reply:
I think it's just used here: https://github.com/regl-project/regl/blob/995ed8c520e4d64beb259253598c4eb338162c32/lib/texture.js#L763-L768

Reply:
Absolutely, would future proof this and avoid confusion for any OffscreenCanvas specific stuff.

Per-spec, they _should_ employ the same API, but I've encountered some specifics where libraries rely on the DOM sizing of the element which isn't part of OffscreenCanvas, regl potentially could have a similar issue in the future.

Reply:
Yeah, admittedly this probably just defense against my ignorance of the API. 😄